tests/memory_tests/src/memory_tests_helper/memory_counter.cpp

```diff
@@ -27,8 +27,8 @@ namespace MemoryTest {
     static PROCESS_MEMORY_COUNTERS getMemoryInfo() {
         static PROCESS_MEMORY_COUNTERS pmc;
         pmc.cb = sizeof(PROCESS_MEMORY_COUNTERS);
-        GetProcessMemoryInfo(GetCurrentProcess(),&pmc, pmc.cb);
-        return pmc;
+        if (!GetProcessMemoryInfo(GetCurrentProcess(), &pmc, pmc.cb))
+            throw std::runtime_error("Can't get system memory values");
```

Comment:
"return pmc" is lost, will cause core dump on windows when run test

Reply:
Good catch! Thanks, will be fixed